CHANGELOG.md

```diff
@@ -207,3 +207,9 @@ All of the material changes in this version were courtesy of @50Wliu. Thanks!
 - Initial release
 - Syntax highlighting for Azure Pipelines files
 - Intellisense for Azure Pipelines files
+
+## 1.247.2
```

Comment:
This should be at the top of the file; also, since this is a minor release, please make sure you follow https://github.com/microsoft/azure-pipelines-vscode/blob/main/RELEASE.md to re-generate the service schema.

Reply:
Hello @50Wliu,

Thank you for reviewing this PR! 

I have re-generated the service schema and corrected the CHANGELOG. I have then triggered the release pipeline against my branch and the artifacts got generated. However, I am unable to push the changes to the [Marketplace](https://marketplace.visualstudio.com/manage/publishers/ms-azure-devops) as I don't seem to have access to it. Can you please tell me how I can get this access?

![image](https://github.com/user-attachments/assets/0dd0ab6f-8853-4100-9a7f-a9535d94977d)

Reference: https://github.com/microsoft/azure-pipelines-vscode/blob/main/RELEASE.md.

Reply:
I don't have access either (I'm not in the ADO org) - @DergachevE, could you please help here?

Reply:
Hello @50Wliu,

@DergachevE got me the required access. 

Step #7 in the [Release process](https://github.com/microsoft/azure-pipelines-vscode/blob/main/RELEASE.md#releasing-the-extension) isn't detailed enough. Does shipping it to Marketplace means uploading the VSIX file by selecting 'update' from the Manage Publishers & Extensions page?

![image](https://github.com/user-attachments/assets/270566b1-e244-4ff3-bd21-93e586321a38)